<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>610</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Demo Vacuum Chamber GUI</string>
    </property>
   </widget>
   <widget class="QLabel" name="pressureTitle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>122</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pressure</string>
    </property>
   </widget>
   <widget class="QLabel" name="pressureTitle_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>185</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>128</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>currentTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPres">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>168</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>currentPres</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTemp">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>200</y>
      <width>168</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>currentTemp</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="presGraph">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="tempGraph">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="presDial">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="tempDial">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="refreshLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>229</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Refresh Interval (ms)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="refreshBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
